backend/homework/4/src/main/java/org/example/question1/StudentUtil.java

```
@@ -0,0 +1,70 @@
+package org.example.question1;
+
+public class StudentUtil {
+
+
+    public static int[] getStudentsByGPA(double lower, double higher, int[] studentIdList, char[][] studentsGrades) {
+        if (lower < 0 || higher < 0 || lower > higher) {
+            return new int[0];
```

Lets return an exception here